src/libraries/System.Reflection.Metadata/src/System/Reflection/Internal/MemoryBlocks/MemoryMappedFileBlock.cs

```diff
@@ -18,7 +18,7 @@ private sealed class DisposableData : CriticalDisposableObject
             public DisposableData(IDisposable accessor, SafeBuffer safeBuffer, long offset)
             {
                 // Make sure the current thread isn't aborted in between acquiring the pointer and assigning the fields.
-#if !NETSTANDARD1_1
+#if FEATURE_CER
```

Comment:
Nit: To conditionally remove a CER, I'd instead do this:

```cs
#if FEATURE_CER
    RuntimeHelpers.PrepareConstrainedRegions();
    try { /* intentionally left blank */ }
    finally
#endif
    {
        /* remainder of logic */
    }
```

Saves the try / finally info from being embedded into the resulting assembly file. It's just minor cleanup, though, and it might even make sense for us to do it as a separate PR. Might even find some other empty try blocks lurking in other .cs files. :)

Reply:
Good suggestion.  I'll go ahead and do that here; there aren't many of them.